package.json

```diff
@@ -143,7 +143,7 @@
   },
   "dependencies": {
     "@chakra-ui/react": "^1.6.6",
-    "@d-i-t-a/reader": "2.0.0-alpha.7",
+    "@d-i-t-a/reader": "^2.0.0-beta.2",
```

Comment:
we should probably not caret this -  we don't want it to automatically pull in potentially unstable updates